templates/semantic-ui/zenodo_rdm/macros/record_item.html

```diff
@@ -30,8 +30,8 @@
 
       {# Top labels #}
       <div class="extra labels-actions">
-        <div class="ui small horizontal primary label">
-          {{ record.ui.publication_date_l10n_long }} ({{ record.ui.version }})
+        <div class="ui small horizontal theme-primary label">
```

Comment:
if you won't have `primary` class, it will break for unstyled communities most likely

Reply:
it is working for non branded ones but just to be safe I'll add it